templates/patterns/navigation_menu/pattern-navigation-menu.html.twig

```diff
@@ -46,5 +46,11 @@
     items: _items,
     icon_path: ecl_icon_path
   } only %}
+{% elseif site_name and ecl_branding == 'standardised' %}
```

Comment:
No need for this, its part of the page layout so it should not be part of the navigation menu.